tests/tls_e2e/host/host.cpp

```diff
@@ -269,7 +269,9 @@ int run_scenarios_tests()
          {false, false, true},
          NEGATIVE_TEST}};
 
+#ifndef _WIN32
     signal(SIGPIPE, SIG_IGN);
+#endif
```

Comment:
Not sure why why this is needed in the linux test but the Windows test passes fine without it, Needs more investigation.

Reply:
winsock treats a broken socket as a normal error and calls will just result in normal WSA errors.